Dockerfiles/midstream/unit_tests.py

```diff
@@ -80,8 +80,9 @@ def test_negative_image_to_test_not_set(self):
                       "Result code not found in %s" % message)
         self.assertEqual(102, result.returncode)
         self.assertTrue(os.path.exists(OUTPUT_DIR+".errormessage"))
-    
-    # Run with a test-operator which passes the bundle image validation job
+
+    # Run with a test-operator which fails the deprecated image check in the
+    # bundle image validation job
```

Comment:
If we change the intention of test-case with the comment then we should check new "expected error" message in some output.